app/src/androidTest/java/com/android/unio/end2end/EndToEndTest.kt

```diff
@@ -98,15 +98,17 @@ open class EndToEndTest : FirebaseEmulatorFunctions {
 
   /* Constant URLs used by the local emulator */
   object Firestore {
-    const val HOST = "10.0.2.2"
+//    const val HOST = "10.0.2.2"
+    const val HOST = "128.179.201.248"
     const val PORT = 8080
     const val ROOT = "http://$HOST:$PORT"
     const val SHORT_ROOT = "$HOST:$PORT"
     const val DATABASE_URL = "$ROOT/emulator/v1/projects/unio-1b8ee/databases/(default)/documents"
   }
 
   object Auth {
-    const val HOST = "10.0.2.2"
+//    const val HOST = "10.0.2.2"
+    const val HOST = "128.179.201.248"
```

Comment:
Don't forget to undo these changes, as they are specific to your case (e.g. running the tests on your phone)